app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="login-cont">
+  <div class="shared_nav">
+  <div class="column">
+     <a class="hamburger" href="http://[::1]:3000/">&#8592;</a>
+     <p>login</p>
+  </div>
+  </div>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %><br />
+      <%= f.password_field :password, autocomplete: "current-password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
```

Comment:
- [ ] As requested by the previous reviewer you should remove the input's label and use placeholders instead your your design to match the design guidelines on the login page.